### ISSUE TITLE
Context is mutated after accessing it in the path

### ISSUE THREAD

**Issue by ruscoder** [HL7/fhirpath.js#155]
Hi. 

I encountered the bug after upgrading from 3.11.0, when I pass the context:
```
const ctx = { a: 1, b: 1 };
fhirpath.evaluate({}, '%a', ctx);
```

`a` gets removed from `ctx`.

I think fhirpath should not mutate the original context.

**Comment by plynchnlm** [comment]
Yes, that is unexpected.  We'll look into it.

**Comment by yuriy-sedinkin** [comment]
Fixed in version 3.13.4.
